Header header
float32[6] setpoint		# servo angle setpoint for all 6 arms [rad]